internal/service/elbv2/load_balancer.go

```diff
@@ -667,6 +667,10 @@ func modifyLoadBalancerAttributes(ctx context.Context, conn *elbv2.ELBV2, arn st
 
 	// Not all attributes are supported in all partitions.
 	for {
+		if len(input.Attributes) == 0 {
+			return nil
+		}
+
```

Comment:
This could go after `input.Attributes = slices.DeleteFunc()` on line 682 if we want this func to continue to fail if an empty slice is passed in, but I see no reason to do so.